packages/runtime/src/widgets/Form/Dropdown.tsx

```diff
@@ -100,7 +100,7 @@ const Dropdown: React.FC<DropdownProps> = (props) => {
   };
 
   const { id, rootRef, values } = useRegisterBindings(
-    { ...rest, initialValue: props.value, selectedValue, widgetName },
+    { ...rest, initialValue: props.value, value: selectedValue, widgetName },
```

Comment:
We need to keep selectedValue binding as well for backwards compatibility, just add the value binding